metabase/metabase
# Adding moderated status to entity picker

Closes https://github.com/metabase/metabase/issues/49398

### Description
This PR exposes an items moderated status in the entity picker in 3 places:
- Search Results
- Recents
- In item lists with browsing

### How to verify

1. Have an instance with some verified question
2. Go to new -> question and find your verified question / model via recents, search, and in the collection tree
3. These icons should not display in OSS

### Demo
![image](https://github.com/user-attachments/assets/7a7152ba-9691-485a-99e4-1212ea6892f4)
### Checklist
